seqr/views/utils/file_utils_2_3_tests.py

```diff
@@ -37,6 +41,12 @@
     ['1', 'NA19678', ''],
 ]
 
+XLSX_PARSED_DATA = [
+    ['Family ID', 'Individual ID', 'Notes'],
+    ['1', 'NA19675', 'An affected individual, additional metadata'],
+    ['1', 'NA19678', None],
```

Comment:
so actually excels parsed data should be the same as everything else and be an empty string, its a bug in `parse_file`. You should change ` padding = [None] * last_col_index` to ` padding = [''] * last_col_index`

Reply:
Okay